docs/articles_en/documentation/openvino-extensibility.rst

```diff
@@ -110,10 +112,6 @@ The ``Identity`` is a custom operation class defined in :doc:`Custom Operation G
          :language: cpp
          :fragment: [add_frontend_extension]
 
-When Python API is used, there is no way to implement a custom OpenVINO operation. Even if custom OpenVINO operation is implemented in C++ and loaded into the runtime by a shared library, there is still no way to add a frontend mapping extension that refers to this custom operation. In this case, use C++ shared library approach to implement both operations semantics and framework mapping.
-
```

Comment:
We lost that part about the inability to add mapping extension for ops loaded from a library if they were not added beforehand when op was implemented. That part is not fixed in the recent version of API, right?

Reply:
returned back